### DISCUSSION TITLE
Correct order of running CIS account baseline.

### DISCUSSION THREAD

**Discussion by vramanufidato** [gruntwork-io/knowledge-base#592] [Help]
We are deploying our AWS architecture using Gruntworks CIS compliance library. We have provisioned using the Reference Architecture and currently we need to setup the baseline-account for mapping the AWS account with the organization.

We are referring but need some clarifications [apply-account-baseline-root-to-root-account](https://docs.gruntwork.io/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/deploy-landing-zone-solution/apply-account-baseline-root-to-root-account)

Do we need to try with account-baseline-root or account-baseline-security ?

Thanks in advance

---

<ins datetime="2022-11-17T11:07:27Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109613">Tracked in ticket #109613</a></p>
</ins>

**Comment by ebeneliason** [comment]
We’d like to help, but we need to understand your question better first. Can you please email support@gruntwork.io so that we can work with you there? We’ll then update this question to be more clear and post our answer here for everyone's benefit. Thanks!
